### PR TITLE
changed NonExportedAmountHelper object into a class

### PR DESCRIPTION
**Pull request by mycp98**
### Description of Work carried through

Brief description of what/why/how.

### CODE REVIEW

File: test/controllers/returns/NonExportedHumanMedicinesPlasticPackagingControllerSpec.scala [hunk 1]
```diff
@@ -105,26 +108,9 @@ class NonExportedHumanMedicinesPlasticPackagingControllerSpec extends PlaySpec w
...
-        any(),
-        ArgumentMatchers.eq(NormalMode),
-        ArgumentMatchers.eq(false),
-        ArgumentMatchers.eq(false)
-      )(any(),any())
-
-    }
-
     "redirect GET to home page when DirectlyExportedComponentsPage and PlasticExportedByAnotherBusinessPage amount not found" in {
+      reset(nonExportedAmountHelper)
```
Comment (lmcewanhmrc) [inline test/controllers/returns/NonExportedHumanMedicinesPlasticPackagingControllerSpec.scala:112]: Need this reset here? (It's already in beforeEach())

Comment (lmcewanhmrc) [reply test/controllers/returns/NonExportedHumanMedicinesPlasticPackagingControllerSpec.scala:112]: (There a few more places too)

**Review by lmcewanhmrc — APPROVED** [review]
Nice, thanks

File: test/controllers/returns/NonExportedHumanMedicinesPlasticPackagingControllerSpec.scala (hunk 1 above)
Comment (mycp98) [reply test/controllers/returns/NonExportedHumanMedicinesPlasticPackagingControllerSpec.scala]: Thank youu just removed the resets that weren’t needed

**Pull request merged**